Doppler.HtmlEditorApi/Repositories.DopplerDb/DopplerCampaignContentRepository.cs

```diff
@@ -71,11 +72,19 @@ public async Task<CampaignState> GetCampaignState(string accountName, int campai
             : campaignStateData.Status == DopplerCampaignStatusInWinnerInABSelectionProcess ? CampaignStatus.InWinnerInABSelectionProcess
             : CampaignStatus.Other;
 
-        return new CampaignState(
-                campaignStateData.OwnCampaignExists,
+        if (campaignStateData.TestType == 0)
+        {
+            return new ClassicCampaignState(campaignStateData.ContentExists, campaignStateData.EditorType, campaignStatus);
+        }
+
+        var campaignTestABCondition = campaignStateData.TestType is DopplerCampaignTestTypeSubject ? TestABCondition.TypeTestABSubject
```

Comment:
Why you are doing different types of comparisons for the same property so near?

![image](https://user-images.githubusercontent.com/1157864/168311425-70aeb9e5-d3ac-43dc-aec4-a0c3ed5af9c0.png)

What do you think about doing simple comparisons in both places?

```suggestion
        var campaignTestABCondition = campaignStateData.TestType == DopplerCampaignTestTypeSubject ? TestABCondition.TypeTestABSubject
```

![image](https://user-images.githubusercontent.com/1157864/168311501-3629dc32-4350-48eb-a688-82c0a13f4b07.png)

Reply:
Fixed!